### PR TITLE
Add yolo-v2-tiny-vehicle-detection-0001 model

### PR DESCRIPTION
**Pull request by LeonidBeynenson**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by LeonidBeynenson** [comment]
2 @vladimir-dudnik:  
Vladimir, please, take a look on this PR.

Please, note that I wrote that the demo object_detection_demo_yolov3_async works with all Yolo V2, Tiny Yolo V2 and Yolo V3 models.
But I have not checked if the demo works with the yolo networks from the AVA team.

**Comment by eizamaliev** [comment]
@LeonidBeynenson please add AC config too.

**Comment by LeonidBeynenson** [comment]
2 @eizamaliev :   Added accuracy checker config

File: tools/accuracy_checker/configs/yolo-v2-tiny-vehicle-detection-0001.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        weights: intel/yolo-v2-tiny-vehicle-detection-0001/FP16-INT8/yolo-v2-tiny-vehicle-detection-0001.bin
+        adapter:
+          type: yolo_v2
+          anchors: "0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828"
+          num: 5
+          coords: 4
+          classes: 80
+
+    datasets:
+      - name: barrier_vehicle_detection_dataset_index_class_2
```
Comment (eaidova) [inline tools/accuracy_checker/configs/yolo-v2-tiny-vehicle-detection-0001.yml:41]: the dataset naming and addition to definitionshs should be descussed offline

Comment (eaidova) [reply tools/accuracy_checker/configs/yolo-v2-tiny-vehicle-detection-0001.yml:41]: ok, resolve it later

**Comment by LeonidBeynenson** [comment]
It turned out that several my github comments were in `pending` state. 🙁    
Now I sent them and applied all the suggested changes

2 @eizamaliev , @eaidova :    
Could you check if everything is OK now? can this PR be merged?

**Pull request merged**